compiler_opt/rl/inlining/imitation_learning_runner.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for running compilation and collect data for behavior cloning."""
+
+import functools
+from absl import app
+from absl import flags
+from absl import logging
+import gin
+
+from compiler_opt.rl.imitation_learning import generate_bc_trajectories_lib
+from compiler_opt.rl.inlining import imitation_learning_config
+from compiler_opt.tools import generate_test_model  # pylint:disable=unused-import
+
+from tf_agents.system import system_multiprocessing as multiprocessing
+
+flags.FLAGS['gin_files'].allow_override = True
+flags.FLAGS['gin_bindings'].allow_override = True
```

Comment:
Where are these flags defined? They should be declared within the entrypoint rather than overriden.

Reply:
I think this is used in the same way as in, e.g., https://github.com/google/ml-compiler-opt/blob/513d50da3662081a0e20ea1222788e8a6718b17d/compiler_opt/tools/generate_default_trace_test.py#L35 
From what I remember I need it because there are gin bindings set by common.gin which I need to overwrite. We had discussed this with Mircea a while back and this was the recommended way to proceed.

Reply:
This is now fixed, the flags are removed as it turns out they were not needed anymore.